jina/peapods/__init__.py

```diff
@@ -12,7 +12,7 @@
     import argparse
 
 
-def Pea(args: 'argparse.Namespace' = None, allow_remote: bool = True, **kwargs):
+def Pea(args: 'argparse.Namespace' = None, allow_remote: bool = True, **kwargs) -> Union['RemotePea', 'ContainerPea', 'HeadPea', 'TailPea', 'BasePea']:
```

Comment:
`BasePea` is the parent class, so by claiming u return `BasePea` you are having a correct statement